docs/reference-guides/best-practices/rancher-server/tips-for-scaling-rancher.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Tips for Scaling Rancher
+---
+
+This guide aims at introducing challenges and approaches that should be considered to scale Rancher setups. As setups grow performance will reduce. There are steps we can take to minimize the load that is put on Rancher and optimize Rancher's ability to handle increased loads.
+
+## General Tips on Optimizing Rancher's Performance
+* It is advisable to keep Rancher up to date with patch releases. Performance improvements and bug fixes are made throughout the life of a minor release. You can review the release notes to help inform your own decisions on whether an upgrade is necessary but we recommend keeping yourself up to date in most cases.
+
+* The geographically closer Rancher's infrastructure and downstream cluster's infrastructure are the better. If a user or organization requires clusters/nodes all over the world or spread across many regions, it is best to use multiple Rancher installations.
+
+## Minimizing Load on the local cluster
+The largest bottleneck when scaling Rancher is kubernetes resource growth in the local cluster. The local cluster contains information for all downstream clusters. Many operations that apply to downstream clusters will create new objects in the local cluster and require computation from handlers running in the local cluster.
+
+### Managing Your Object Counts
+ETCD eventually encounters limitations to the number of a single kubernetes resource type it can store. These exact numbers are not well documented. From internal observations we usually see performance issues once an object count exceeds 60k. Usually the kubernetes resource type that reaches these numbers is Rolebinding.
+
+Rolebindings are created in the local cluster as a side effect of many operations. This includes creating clusters, adding users, adding projects, and adding roles to users. When adding users to cluster or projects, rolebindings will be created.
+
+Considerations when attempting reduce rolebindings in the local cluster:
+* Only add users to clusters and projects when necessary
+* Remove clusters and projects when they are no longer needed
+* Only use custom roles if necessary
+* Use as few rules as possible in custom roles
+* Consider whether adding a role to a user is redundant
+* Consider that using less, but more powerful, clusters may be more efficient
+* Monitoring the rolebindings created from creating new projects for specific purposes versus creating new clusters
+
+### Using Apps V2 Over Apps V1
+There are two App kubernetes resources that Rancher uses: apps.projects.cattle.io and correct this apps.cattle.cattle.io. V1 here refers to apps.projects.cattle.io since they were introduced first and are legacy, while V2 refers to apps.catalog.cattle.io. Apps V2 are preferrable because they live in the downstream cluster while Apps V1 lives in the local cluster. The dashboard uses Apps V2 while the old `/g/clusters` UI uses Apps V1.
```

Comment:
You should use `TODO` :) 

```suggestion
There are two Application resources that Rancher uses: apps.projects.cattle.io and apps.cattle.cattle.io. V1 here refers to apps.projects.cattle.io since they were introduced first and are legacy, while V2 refers to apps.catalog.cattle.io. Apps V2 are preferrable because they live in the downstream cluster while Apps V1 lives in the local cluster. The dashboard uses Apps V2 while the old `/g/clusters` UI uses Apps V1.
```

Is this worth including since we are already migrating away from cluster manager ? It would be simpler if we just say "use cluster explorer apps", than listing the CRD types.

Reply:
The CRD is literally called App(s). I think Applications would be misleading. I could included that it is short for applications? Also what do you mean by using TODO? Yes I think it's relevant since a lot of scaled users have used rancher for a while and we don't know exactly what they are doing, especially because we have not made a migration path for Apps public.

Reply:
You had this, which I assumed was a todo: 
>  and correct this apps.cattle.cattle.io

Even if we want to tell people this, do we think giving them the CRD's is the best course of action ? What are you hoping they do with this information, to get a count of both CRD types and see which is larger ? Or just migrate old apps to explorer apps. If we keep this info I would like to rewrite it.

Reply:
@rmweir I think I agree with @cbron - if I read between the lines correctly what we want to do is be prescriptive about what users should and should not do.

So here's a question for you: what can the user do in order to minimize V1 resource use and/or maximize V2 resource use? Is avoiding to use `/g/clusters` all they can do or is there more?